flake.nix

```diff
@@ -274,8 +274,15 @@
                     fi
                   fi
 
+                  export RUSTC_WRAPPER=${pkgs.sccache}/bin/sccache
+                  export CARGO_BUILD_TARGET_DIR="''${CARGO_BUILD_TARGET_DIR:-''${root}/target-nix}"
+                  export FM_DISCOVER_API_VERSION_TIMEOUT=10
+                  [ -f "$root/.shrc.local" ] && source "$root/.shrc.local"
+
                   if [ ''${#TMPDIR} -ge 40 ]; then
-                      >&2 echo "⚠️  TMPDIR too long. This might lead to problems running tests and regtest fed. Are you nesting 'nix develop' invocations?"
+                      >&2 echo "⚠️  TMPDIR too long. This might lead to problems running tests and regtest fed. Will try to use /tmp/ instead"
+                      # Note: this will only work `nix develop`, not in a dev shell?
```

Comment:
Still an open question?

Reply:
Yeah, it doesn't work for me (but I don't need it, so whatevs).